harshita/level1/bonus.cpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+    double year,salary,amount=0;
+    cout << "Numbers of years : ";
+    cin >> year ;
+    cout << "Enter salary: ";
+    cin >> salary ;
+    if(year > 5){
+        amount=salary*0.05;
+        cout << amount << " bonus... yey you got bonus!";
+        
+    }
+
+    else{
```

Comment:
It should be `cout << "No Bonus";` No need to print amount. Here we have to print the bonus. So I would recommend use a bonus as a variable rather than an amount. Logic is right.